packages/cube/package.json

```
@@ -0,0 +1,49 @@
+{
+  "name": "barnard59-cube",
+  "version": "2.0.0",
+  "description": "RDF cubes for Linked Data pipelines",
+  "type": "module",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/zazuko/barnard59.git",
+    "directory": "packages/cube"
+  },
+  "keywords": [],
+  "author": "Thomas Bergwinkl <bergi@axolotlfarm.org> (https://www.bergnet.org/people/bergi/card#me)",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/zazuko/barnard59/issues"
+  },
+  "homepage": "https://github.com/zazuko/barnard59",
+  "peerDependencies": {
+    "barnard59-base": "^2.0.0",
+    "barnard59-formats": "^2.0.0",
+    "barnard59-http": "^2.0.0",
+    "barnard59-rdf": "^2.0.0",
+    "barnard59-sparql": "^2.0.0",
+    "barnard59-validate-shacl": "^0.3.8"
+  },
+  "engines": {
+    "node": ">= 14.0.0"
+  },
+  "dependencies": {
+    "@zazuko/env-node": "^1.0.0",
+    "external-merge-sort": "^0.1.2",
+    "lodash": "^4.17.21",
+    "rdf-dataset-ext": "^1.0.1",
```

Please use `this.env` provided by step factories (re #195). Check the snippets in #198

```suggestion
    "external-merge-sort": "^0.1.2",
    "lodash": "^4.17.21",
```

---

I would be more confident if you could apply these changes